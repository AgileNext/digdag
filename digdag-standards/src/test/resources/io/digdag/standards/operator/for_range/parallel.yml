_command:
  from: 0
  until: 4
count: 4
_parallel: true
_do:
  echo>: do
