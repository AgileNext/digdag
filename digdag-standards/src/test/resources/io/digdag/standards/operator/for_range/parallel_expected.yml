+range-from=0&until=1:
  echo>: do
  _export:
    range: {from: 0, until: 1}
+range-from=1&until=2:
  echo>: do
  _export:
    range: {from: 1, until: 2}
+range-from=2&until=3:
  echo>: do
  _export:
    range: {from: 2, until: 3}
+range-from=3&until=4:
  echo>: do
  _export:
    range: {from: 3, until: 4}
_parallel: true
